.gitignore

```diff
@@ -1,5 +1,5 @@
 build/
-build*/
+build-fprime-*/
```

Comment:
This needs to include `build-artifacts` as well.

I'd recommend:

```
build-fprime-*/
build-artifacts/
```
or something similar

Reply:
good point – adding in